src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -113,6 +113,10 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
     this.props.onClick(this.props.data.startTime);
   };
 
+  private _handleExpand = (e: MouseEvent) => {
+    e.stopPropagation();
```

Comment:
can this be handled inside `ExpandableText` component? what if another comp will use `ExpandableText` and need to stop propagation?

Reply:
that's same issue we had with A11yWrapper that by default made stopPropagation. We reach cases when event should be propagated, so we moved stopPropagation outside of A11yWrapper and allows plugins manage event propagation by itself.  In one hand it makes some code-duplication but on another hand we got more flexible solution.